content/reference/rest-api/engine-diagrams/create.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Engine Diagrams: create'
+weight: 10
+
+menu:
+  main:
+    name: 'create'
+    identifier: 'rest-api-engine-diagrams-create'
+    parent: 'rest-api-engine-diagrams'
+    pre: 'Creates or updates an engine diagram.'
+---
+
+Creates a new engine diagram if it doesn't exist yet or updates an engine diagram with a new version.
+
+# Request
+
+## HTTP Request
+
+```
+PUT https://cawemo.com/api/v1/engine-diagrams
+```
+
+## Request Body
+
+In the request body, supply an [Engine Diagrams resource]({{< ref "/reference/rest-api/engine-diagrams/_index.md#resource-representation" >}}) with the following properties:
+
+<table class="table table-striped">
+  <tr>
+    <th>Property Name</th>
+    <th>Type</th>
+    <th>Description</th>
+    <th>Notes</th>
+  </tr>
+  <tr>
+    <td>content</td>
+    <td>string</td>
+    <td>The engine diagram's BPMN 2.0 XML definition.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>created (optional)</td>
+    <td>date (UNIX Epoch time in milliseconds)</td>
+    <td>The creation date assigned to the engine diagram's version.</td>
+    <td>Has to lie in the past. Defaults to <i>now</i>.</td>
+  </tr>
+  <tr>
+    <td>version</td>
+    <td>number</td>
+    <td>The version of the engine diagram.</td>
```

Comment:
'The version of the process definition.' would be more aligned with the platform docs: https://docs.camunda.org/manual/7.15/reference/rest/process-definition/get/#result

Reply:
If we want to use `process definition`, then we should replace `engine diagram` with this term everywhere.

Reply:
As agreed on in today's meeting we'll go with `process definition` instead of `engine diagram` everywhere.